spark-publish/index.js

```diff
@@ -65,7 +66,10 @@ export const publish = async ({
   start = new Date()
   const tx = await ieContract.addMeasurements(cid.toString())
   logger.log('Waiting for the transaction receipt:', tx.hash)
-  const receipt = await tx.wait()
+  const receipt = await pRetry(() => tx.wait(1 /* confirmation(s) */, 120_000 /* 2 minutes */), {
+    onFailedAttempt: err => console.error(err),
+    maxRetryTime: 600_000 // 10-minute timeout
+  })
```

Comment:
I find this a bit hard to read with the two inline comments, what do you think about this?

```suggestion
  const receipt = await pRetry(
    () => tx.wait(
      1, // confirmation(s)
      120_000 // 2 minutes
    ), {
      onFailedAttempt: err => console.error(err),
      maxRetryTime: 600_000 // 10-minute timeout
    }
  )
```

Reply:
Yeah, I had similar concerns. Thank you for the suggestion!